migrations/update_2_3_0/index.ts

```diff
@@ -0,0 +1,12 @@
+import { Rule, SchematicContext, Tree } from "@angular-devkit/schematics";
+
+const version = "2.3.0";
+
+export default function (): Rule {
+  return (tree: Tree, context: SchematicContext) => {
+    context.logger.info(
```

Comment:
Since this is just to enable migrations in the first place, you can probably skip the message. Fairly certain the `Rule` can be just a void function too